Index: grails-app/services/xyz/ekkor/ArticleSrvice.groovy
===================================================================
--- grails-app/services/xyz/ekkor/ArticleSrvice.groovy	(date 1556373307000)
+++ grails-app/services/xyz/ekkor/ArticleSrvice.groovy	(date 1556373307000)
@@ -1,19 +0,0 @@
-package xyz.ekkor
-
-import grails.gorm.transactions.Transactional
-import groovy.transform.CompileStatic
-
-@CompileStatic
-@Transactional
-class ArticleSrvice {
-
-    ArticleDataService articleDataService
-
-    @Transactional
-    Article save(Article articleInstance, Avatar author, Category category) {
-        Article article = articleDataService.save(articleInstance, author, category)
-
-    }
-
-
-}
Index: grails-app/services/xyz/ekkor/FIleService.groovy
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- grails-app/services/xyz/ekkor/FIleService.groovy	(date 1556373307000)
+++ grails-app/services/xyz/ekkor/FIleService.groovy	(date 1556509239000)
@@ -1,6 +1,6 @@
 package xyz.ekkor
 
-import grails.transaction.Transactional
+import grails.gorm.transactions.Transactional
 
 @Transactional
 class FIleService {
Index: grails-app/services/xyz/ekkor/MainService.groovy
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- grails-app/services/xyz/ekkor/MainService.groovy	(date 1556373307000)
+++ grails-app/services/xyz/ekkor/MainService.groovy	(date 1556509239000)
@@ -1,8 +1,8 @@
 package xyz.ekkor
 
+import grails.gorm.transactions.Transactional
 import grails.plugin.cache.Cacheable
 import grails.plugin.springsecurity.SpringSecurityService
-import grails.transaction.Transactional
 import org.hibernate.FetchMode
 
 @Transactional
Index: grails-app/services/xyz/ekkor/NotificationService.groovy
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- grails-app/services/xyz/ekkor/NotificationService.groovy	(date 1556373307000)
+++ grails-app/services/xyz/ekkor/NotificationService.groovy	(date 1556509239000)
@@ -1,6 +1,6 @@
 package xyz.ekkor
 
-import grails.transaction.Transactional
+import grails.gorm.transactions.Transactional
 
 @Transactional
 class NotificationService {
Index: grails-app/services/xyz/ekkor/SanitizeService.groovy
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- grails-app/services/xyz/ekkor/SanitizeService.groovy	(date 1556373307000)
+++ grails-app/services/xyz/ekkor/SanitizeService.groovy	(date 1556509239000)
@@ -1,6 +1,6 @@
 package xyz.ekkor
 
-import grails.transaction.Transactional
+import grails.gorm.transactions.Transactional
 import org.owasp.html.AttributePolicy
 import org.owasp.html.PolicyFactory
 import org.owasp.html.Sanitizers
Index: grails-app/services/xyz/ekkor/EncryptService.groovy
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- grails-app/services/xyz/ekkor/EncryptService.groovy	(date 1556373307000)
+++ grails-app/services/xyz/ekkor/EncryptService.groovy	(date 1556509239000)
@@ -1,6 +1,7 @@
 package xyz.ekkor
 
-import grails.transaction.Transactional
+import grails.converters.JSON
+import grails.gorm.transactions.Transactional
 
 import javax.crypto.Cipher
 import javax.crypto.SecretKeyFactory
Index: grails-app/services/xyz/ekkor/UserService.groovy
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- grails-app/services/xyz/ekkor/UserService.groovy	(date 1556373307000)
+++ grails-app/services/xyz/ekkor/UserService.groovy	(date 1556509239000)
@@ -1,6 +1,7 @@
 package xyz.ekkor
 
-import grails.transaction.Transactional
+import grails.gorm.transactions.Transactional
+
 
 @Transactional
 class UserService {
